scipy/signal/tests/test_filter_design.py

```diff
@@ -1876,14 +1887,16 @@ def test_output_type(self):
                 assert isinstance(p, np.ndarray)
 
     @pytest.mark.parametrize(
-        'func', [buttap, besselap,
-                 lambda N, xp: cheb1ap(N, 1, xp=xp),
-                 lambda N, xp: cheb2ap(N, 20, xp=xp),
-                 lambda N, xp: ellipap(N, 1, 20, xp=xp)
-                ],
+        'base_func,func', [
+            make_xp_pytest_param(buttap, buttap),
```

Comment:
I'm sure there's a deep reason for `make_xp_test_param` requiring two arguments. Not sure what it is though.

Reply:
Agreed about documentation. The next PR I plan to submit is for exactly that.